gdspy/__init__.py

```diff
@@ -3411,7 +3411,7 @@ def read_gds(self,
                  layers={},
                  datatypes={},
                  texttypes={},
-                 prefix=''):
+                 rename_template='{name}'):
```

Comment:
`layer`, `datatype` and `texttype` should be the last kwargs. Include this after `rename` — the tests can be modified to follow the new arguments.